packages/geocoder/src/index.test.js

```diff
@@ -19,6 +19,10 @@ describe("geocoder", () => {
       baseUrl: "https://ws-st.trimet.org/pelias/v1",
       type: "PELIAS"
     },
+    {
```

Comment:
This file should have `ts` extension too.

Reply:
Making this file typed would create a lot of work to fix all the mocks and other things to use correct data structures when it's not necessary. Is there a better way of typing a file like this?

Reply:
Does just changing the extension to `.ts` cause TypeScript errors? (TypeScript can infer types and will enforce only where other code requires them).